src/main/java/core/basesyntax/helpers/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.helpers;
+
+import core.basesyntax.enums.Colors;
+import core.basesyntax.enums.Figures;
+import core.basesyntax.interfaces.RandomNumber;
+import core.basesyntax.models.Circle;
+import core.basesyntax.models.Figure;
+import core.basesyntax.models.IsoscelesTrapezoid;
+import core.basesyntax.models.Rectangle;
+import core.basesyntax.models.RightTriangle;
+import core.basesyntax.models.Square;
+
+public class FigureSupplier implements RandomNumber {
+    private static final int MAX_FIGURE_SIZE = 100;
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
```
making it an instance field is advisable, this will make your class more testable and flexible

Reply:
++